frontend/src/components/ForgotPassword/ForgotPassword.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { FormControl, TextField, Button, Typography } from '@mui/material';
+
+import {
+  TextFieldWrap,
+  WrapH1,
+  BorderForm,
+  RegistrationFormWrapper,
+  StyledSpan,
+  WrapButtonAndText
+} from './styles';
+
+function ForgotPassword() {
+  return (
+    <RegistrationFormWrapper>
+      <BorderForm>
+        <FormControl sx={{ width: '35ch' }}>
+          <WrapH1>
+            <Typography>Forgot password?</Typography>
+          </WrapH1>
+
+          <TextFieldWrap>
+            <TextField
+              placeholder="Please enter your email"

```

Comment:
placeholder doesn't work in MUI

instead use prop label and FormHelperText comp to show warnings or errors

Reply:
you can also use prop fullWidth to stretch to make textfield a full size

Reply:
done